src/main/java/com/rhombus/api/examples/EnvironmentalKillSwitch.java

```diff
@@ -0,0 +1,169 @@
+        final long hot = Integer.parseInt(hotString);
+        final long cold = Integer.parseInt(coldString);
+        final String alias = commandLine.getOptionValue("alias");
+        final String host = commandLine.getOptionValue("host");
+
+        _initialize(apikey);
+
+        Boolean running = true;
+        while (running)
+        {
+            float celsius = climateData();
+            long fahrenheit = CelsiusConvertToFahrenheit(celsius);
+            if (fahrenheit > hot)
+            {
+                kill(alias, host, plug);
+            }
+            else if (fahrenheit < cold)
+            {
+                on(alias, host, plug);
+            }
+            Thread.sleep(1000);
+        }
+
+    }
+
+    private static void _initialize(final String apiKey) throws Exception
+    {
+        /*
+         * API CLIENT
+         */
+        {
+            final List<Header> defaultHeaders = new ArrayList<>();
+            defaultHeaders.add(new BasicHeader("x-auth-scheme", "api-token"));
+            defaultHeaders.add(new BasicHeader("x-auth-apikey", apiKey));
+
+            final ClientBuilder clientBuilder = ClientBuilder.newBuilder();
+            clientBuilder.register(new JacksonJaxbJsonProvider().configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false).configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false));;
+
+            final HostnameVerifier hostnameVerifier = new HostnameVerifier() {
+                @Override
+                public boolean verify(String hostname, SSLSession session) {
+                    return true;
+                }
+            };
+
+            clientBuilder.hostnameVerifier(hostnameVerifier);
+
+            _apiClient = new ApiClient();
+            _apiClient.setHttpClient(clientBuilder.build());
+            _apiClient.addDefaultHeader("x-auth-scheme", "api-token");
+            _apiClient.addDefaultHeader("x-auth-apikey", apiKey);
+
+            _climatewebservice = new ClimateWebserviceApi(_apiClient);
+
+        }
+    }
+
+    public static void kill(String alias, String host, int plug) throws Exception
+    {
+        if (alias != null)
+        {
+            Runtime.getRuntime().exec("kasa --strip --alias "  + alias +  " off --index "  + Integer.toString(plug -1));
+        }
+        else if (host != null)
+        {
+            Runtime.getRuntime().exec("kasa --strip --host "  + host +  " off --index "  + Integer.toString(plug -1));
+        }
+    }
+
+    public static void on(String alias, String host, int plug) throws Exception
+    {
+        if (alias != null)
+        {
+            Runtime.getRuntime().exec("kasa --strip --alias "  + alias +  " on --index "  + Integer.toString(plug -1));
+        }
+        else if (host != null)
+        {
+            Runtime.getRuntime().exec("kasa --strip --host "  + host +  " on --index "  + Integer.toString(plug -1));
+        }
+    }
+
+    public static float climateData() throws Exception
```

Comment:
I'm not sure what this is actually returning here.  It looks like it is just looking at the last climate sensor data (which can be somewhat random since the order of the minimal climate state list is not guaranteed).  You should include a required cli argument for the desired climate sensor uuid (or name).